tests/dialects/test_dmp.py

```diff
@@ -0,0 +1,77 @@
+from xdsl.dialects.experimental.dmp import ExchangeDeclarationAttr, ShapeAttr
+from xdsl.transforms.experimental.dmp import decompositions
+
+
+def flat_face_exchanges(
+    shape: ShapeAttr, dim: int
+) -> tuple[ExchangeDeclarationAttr, ExchangeDeclarationAttr]:
+    func = (
+        decompositions._flat_face_exchanges_for_dim  # pyright: ignore[reportPrivateUsage]
```

Comment:
Can you please add a small comment saying why you need to use a private function here?

Reply:
I mean even if it's a test file it might still be a useful thing to add